code/src/java/gmgen/plugin/PcgSystemInitiative.java

```diff
@@ -20,14 +20,15 @@
  */
 package gmgen.plugin;
 
-import gmgen.plugin.dice.Dice;
 import pcgen.cdom.util.CControl;
 import pcgen.cdom.util.ControlUtilities;
 import pcgen.core.Globals;
 import pcgen.core.PCStat;
 import pcgen.core.PlayerCharacter;
 import pcgen.core.display.CharacterDisplay;
 
+import gmgen.plugin.dice.Dice;
```

Comment:
Your config file currently reads gmgem not gmgen

Reply:
I don't see this GMGem reference here... please verify?